pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.builds

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="..\..\dir.props" />
```

Comment:
It's confusing because we had a system where only one project used the new packaging model (Microsoft.NETCore.App), and it self-contained everything, then I moved that into a common location, but there are still two packages that don't produce runtime packages and are on the old model.  I think I can try to make them all play nicely though.